bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -6,15 +6,24 @@
 
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}
-{% if l10n_has_tag('fx49_remove_hello') %}
+{% if l10n_has_tag('firefox_title_meta_bug1345338') %}
+  {{ _('Free Web Browser for Android, iOS, Desktop | Firefox') }}
+{% elif l10n_has_tag('fx49_remove_hello') %}
   {{ _('Firefox – Desktop browser, Android, iOS, Sync, Private Browsing') }}
 {% elif l10n_has_tag('tracking_protection') %}
   {{ _('Firefox – Desktop browser, Android, iOS, OS, Hello, Sync, Private Browsing') }}
 {% else %}
   {{ _('Firefox — Desktop browser, Android, OS, Hello, Sync, Marketplace') }}
```

Comment:
Not specifically part of this change, but can you check with Peying to see if we can remove these two fallbacks that mention Hello? These shouldn't be used anymore (for obvious reasons 😁).